commander/commander.go

```diff
@@ -97,6 +97,11 @@ func (c *Commander) Start() (err error) {
 		return err
 	}
 
+	err = c.storage.MigratePubKeyPendingState()
+	if err != nil {
+		return err
+	}
```

Comment:
a little strange to add a migration when there's only one active database, once this is deployed the migration can be dropped